docs/website/docs/user-guides/advanced/connecting-a-service.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Connecting to a service
+sidebar_position: 1
+---
+
+This tutorial will show you how you can connect your Go application to a mongodb service.
+
+Building on top of the [Go quickstart guide](../quickstart/go.md), we will extend the application to check if it is connected to a mongodb service.
+
+### Pre-requisites
+1. Install Service Binding Operator on your cluster.  See [installation instruction](https://operatorhub.io/operator/service-binding-operator).
+
+2. Install Percona Server Mongodb operator on your cluster. See [installation instruction](https://operatorhub.io/operator/percona-server-mongodb-operator).
+:::note
+The operator will be installed in a new namespace called "my-percona-server-mongodb-operator" and will be usable from that namespace only.
+:::
+3. Create a mongodb service.
+
+import CreateMongodbService from './_create-mongodb-service.mdx';
+
+<CreateMongodbService/>
+
+### Implement the code logic
+:::note
+If you're already running `odo dev` in a terminal, exit it and start afresh.
+:::
+
+The new code is simple. We obtain the connection information (username, password, and host) from the environment, and use it to connect to the mongodb service and ping it. 
```

Comment:
```suggestion
The below code obtains the connection information (username, password, and host) from the environment. We then use it to connect to the MongoDB service and query it. 
```

Maybe this will be better?

Reply:
I'd stick to _ping_, since that's what we're really doing in the code. _Query_ sounds more like we're querying something from the database which we are not in this case.